Source/Krypton Components/Krypton.Toolkit/General/ShadowManager.cs

```diff
@@ -7,9 +7,7 @@
  */
 #endregion
 
-// To get around bug in .NET 8, Preview 7
-// TODO: Remove when .NET 8 is GA
-#if NET8_0
+#if NET8_0_OR_GREATER
```

Comment:
Was the `bug` not fixed, or is this just a blind change ?

Reply:
No, there must have had been changes between .NET 7 & 8 in `MethodInvoker`. Removing will cause the build to fail.